homeassistant/helpers/config_validation.py

```diff
@@ -297,9 +297,12 @@ def icon(value: Any) -> str:
             "minutes": vol.Coerce(int),
             "seconds": vol.Coerce(int),
             "milliseconds": vol.Coerce(int),
+            "microseconds": vol.Coerce(int),
         }
     ),
-    has_at_least_one_key("days", "hours", "minutes", "seconds", "milliseconds"),
+    has_at_least_one_key(
+        "days", "hours", "minutes", "seconds", "milliseconds", "microseconds"
```

Comment:
Why are we adding microseconds ? And how is that related to adding collections to template? Please split out this change for a new PR.